images/capi/packer/config/containerd.json

```diff
@@ -1,6 +1,6 @@
 {
-    "containerd_version": "1.5.4",
-    "containerd_sha256": "591e4e087ea2f5007e6c64deb382df58d419b7b6922eab45a1923d843d57615f",
+    "containerd_version": "1.5.5",
+    "containerd_sha256": "45f02cfc65db47cf088c95555906e1dcba7baf5a3fbad3d947dd6b9af476a144",
```

Comment:
where do you get this sha256 from?

as we download the [`cri-containerd-cni`](https://github.com/c445/image-builder/blob/c445/images/capi/packer/qemu/packer.json#L11) binary we must use `45f02cfc65db47cf088c95555906e1dcba7baf5a3fbad3d947dd6b9af476a144` as sha256

Reply:
this is exactly the sha which is set?! It is from https://github.com/containerd/containerd/releases/download/v1.5.5/cri-containerd-cni-1.5.5-linux-amd64.tar.gz.sha256sum

Reply:
:see_no_evil: sorry for confusing you ... not sure where my minds already were when writing this comment